package.json

```diff
@@ -45,7 +45,7 @@
     "pre-build": "yarn check-gb-unused-babelrc && yarn bypass-gb-babel",
     "test:pre-build": "yarn check-gb-unused-babelrc && yarn test:bypass-gb-babel",
     "start": "yarn pre-build && yarn react-native start",
-    "start:reset": "yarn clean:runtime && yarn start --reset-cache",
+    "start:reset": "yarn clean:metro && yarn start --reset-cache",
```

Comment:
There's #103 which looks like it's also taking care of this. Can you maybe review that one and get it merged? Otherwise, I think there will be a conflict